script.js

```diff
@@ -442,14 +498,23 @@ function handleCitiesLayer(geojson) {
 
     // Render the template with all of the properties. Mustache ignores properties
     // that aren't used in the template, so this is fine.
+    const { municipality, state, Country } = layer.feature.properties; 
+    const props = {
+      // Build city name with state and country if supplied
+      jurisdictionName: `${municipality}${state ? `, ${state}`: ''}${Country ? `, ${Country}` : ''}`,
+      jurisdictionType: 'City',
+      ...layer.feature.properties,
+    };
+
     const renderedInfo = Mustache.render(
       infowindowTemplate,
-      layer.feature.properties
+      props
     );
     document.getElementById(
       "aemp-infowindow-container"
     ).innerHTML = renderedInfo;
-    return Mustache.render(popupTemplate, layer.feature.properties);
+    // Override jurisdiction name for popup
+    return Mustache.render(popupTemplate, { ...props, jurisdictionName: municipality });
```

Comment:
I would prefer to just make a different prop of `popupName`, may do this tomorrow, to avoid mutating

Reply:
Done